services/brig/test/integration/API/User/Auth.hs

```diff
@@ -52,12 +57,29 @@ tests conf m z b = testGroup "auth"
                 , test m "failure-suspended" (testSuspendedSsoLogin b)
                 , test m "failure-no-user" (testNoUserSsoLogin b)
                 ]
+            , testGroup "legalhold-login"
+                [ test m "failure-no-team" (testRegularUserLegalHoldLogin b)
+                , test m "team-user-with-legalhold-enabled" (testTeamUserLegalHoldLogin b g)
+                , test m "failure-suspended" (testSuspendedLegalHoldLogin b g)
+                , test m "failure-no-user" (testNoUserLegalHoldLogin b)
+                , test m "failure-wrong-password" (testWrongPasswordLegalHoldLogin b g)
+                , test m "always-persistent-cookie" (testLegalHoldSessionCookie b g)
+                , test m "legalhold-logout" (testLegalHoldLogout b g)
+                ]
+            , testGroup "nginz"
+                [ test m "nginz-login" (testNginz b n)
+                , test m "nginz-legalhold-login" (testNginzLegalHold b g n)
+                ]
```

Comment:
Are [these](https://github.com/wireapp/wire-server/blob/dbf96139ac4d19db854f72abd98306e5679f30a2/services/galley/test/integration/API/Teams/LegalHold.hs#L102-L111) covered now?  If so, please remove that TODO.  If not, please fix!  :)

Reply:
The first one, yes. I don't know about the state of the other two, but they're not related to this PR directly.